lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -361,6 +361,24 @@ public function getClobTypeDeclarationSQL(array $field)
         return 'CLOB';
     }
 
+    /**
+     * Returns the database condition for querying the schema.
+     *
+     * @param string $database The database schema. If NULL or '/', the currently active schema will be queried.
+     *
+     * @return string
```

Comment:
Please remove useless (duplicated) `@return`, it doesn't add any extra information.